test-tools/proof-explorer/package.json

```diff
@@ -5,7 +5,7 @@
     "packageManager": "yarn@4.0.1",
     "dependencies": {
         "@concordium/browser-wallet-api-helpers": "^3.0.0",
-        "@concordium/web-sdk": "^7.1.0",
+        "@concordium/web-sdk": "^7.5.0",
```

Comment:
I suggest to bump the version of the `proof-explorer` so we can re-deploy it and update its changelog.

Reply:
👌🏻